manifests/0000_90_kube-apiserver-operator_03_servicemonitor.yaml

```diff
@@ -44,14 +44,14 @@ spec:
   groups:
   - name: cluster-version
     rules:
-    - alert: TechPreviewNoUpgrade
+    - alert: UnsupportedFeatureSetEnabled
       annotations:
-        summary: Cluster has enabled tech preview features that will prevent upgrades.
+        summary: Cluster has enabled unsupported features that will prevent upgrades.
         description: >-
-          Cluster has enabled Technology Preview features that cannot be undone and will prevent upgrades.
-          The TechPreviewNoUpgrade feature set is not recommended on production clusters.
+          Cluster has enabled a feature set that cannot be undone and will prevent upgrades.
```

Comment:
Can `CustomNoUpgrade` be unset?  I forget where the guard that prevents unsetting `TechPreviewNoUpgrade` lives, but maybe it needs to be updated to consider `CustomNoUpgrade` as well?